src/usbrecord.h

```diff
@@ -17,6 +17,8 @@
 #define RECORD_LENGTH   5
 #define RECORD_SUMMARY  6
 
+class USBPacket;
```

Comment:
also, I dont really like the fact that USBRecord depends on USBPacket, considering that USBPacket already inherits from USBRecord

Reply:
Dependency removed in https://github.com/lambdaconcept/usb2sniffer-qt/pull/27/commits/89523ec40e9990e5afb8ca6b0c154420675f9c7f.